pkg/proxy/proxy.go

```diff
@@ -412,7 +412,7 @@ func (p *Proxy) ReinstallRoutingRules() error {
 			return err
 		}
 
-		if !option.Config.EnableIPSec || option.Config.TunnelingEnabled() {
+		if !option.Config.EnableIPSec {
 			if err := removeFromProxyRoutesIPv4(); err != nil {
 				return err
 			}
```

Comment:
It might make sense to refactor this into:

```
if (foo)
	installFromProxyRoutes()
else
	removeFromProxyRoutes()
```

But I believe you actually need the routes regardless of whether IPsec is enabled, correct?

Reply:
Yes, that is correct, just a small note that IPSec + Ingress/Gateway API are not compatible due to KPR/nodeport requirement.

Reply:
right! So for the time being, should we spell this out as

```
if (Ingress || IPsec with native-routing)
	installFromProxyRoutes()
else
	removeFromProxyRoutes()
```

(if there's no easy way to express "Ingress enabled", then let's say "`!IPsec || IPsec with native-routing`" instead?)